libcnb-cargo/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "libcnb-cargo"
+version = "0.1.0"
+edition = "2021"
```

Comment:
We need to add a number of other fields here - not all of the ones in the other crates, but a fair few - eg `license`, `include` (so the license gets bundled), `repository` etc.

Though these fields are also missing from `libcnb-proc-macros`, so happy to defer to a future PR where we standardise on the fields to use across all crates. Also we should decide what to do for README for the ancillary crates. Perhaps they should not use the repo root README (and only `libcnb` use that), and instead have a shorter readme that links to `libcnb`.

Reply:
I'll add them in this PR for the new crate and create an issue for `libcnb-proc-macros`: #208

Reply:
This has been fixed! :)